pontoon/base/models/user.py

```diff
@@ -220,6 +224,36 @@ def has_approved_translations(self):
     return self.translation_set.filter(approved=True).exists()
 
 
+@property
+def badges_translation_count(self):
+    """Contributions provided by user that count towards their badges."""
+
```

Comment:
Nit: Please remove this blank line (and in the following two functions, too). :)